src/plugins/i18n.ts

```diff
@@ -136,193 +134,179 @@ function i18nMoneyFormatter(amount: any): string {
+        "arenaFlyout",
+        "arenaTag",
+        "battle",
+        "battleScene",
+        "battleInfo",
+        "battleMessageUiHandler",
+        "battlePokemonForm",
+        "battlerTags",
+        "berry",
+        "bgmName",
+        "biome",
+        "challenges",
+        "commandUiHandler",
+        "common",
+        "achv",
+        "dialogue",
+        "battleSpecDialogue",
+        "miscDialogue",
+        "doubleBattleDialogue",
+        "egg",
+        "fightUiHandler",
+        "filterBar",
+        "filterText",
+        "gameMode",
+        "gameStatsUiHandler",
+        "growth",
+        "menu",
+        "menuUiHandler",
+        "modifier",
+        "modifierType",
+        "move",
+        "nature",
+        "pokeball",
+        "pokedexUiHandler",
+        "pokemon",
+        "pokemonCategory",
+        "pokemonEvolutions",
+        "pokemonForm",
+        "pokemonInfo",
+        "pokemonInfoContainer",
+        "pokemonSummary",
+        "saveSlotSelectUiHandler",
+        "settings",
+        "splashMessages",
+        "starterSelectUiHandler",
+        "statusEffect",
+        "terrain",
+        "titles",
+        "trainerClasses",
+        "trainersCommon",
+        "trainerNames",
+        "tutorial",
+        "voucher",
+        "weather",
+        "partyUiHandler",
+        "modifierSelectUiHandler",
+        "moveTriggers",
+        "runHistory",
+        "mysteryEncounters/mysteriousChallengers",
+        "mysteryEncounters/mysteriousChest",
+        "mysteryEncounters/darkDeal",
+        "mysteryEncounters/fightOrFlight",
+        "mysteryEncounters/slumberingSnorlax",
+        "mysteryEncounters/trainingSession",
+        "mysteryEncounters/departmentStoreSale",
+        "mysteryEncounters/shadyVitaminDealer",
+        "mysteryEncounters/fieldTrip",
+        "mysteryEncounters/safariZone",
+        "mysteryEncounters/lostAtSea",
+        "mysteryEncounters/fieryFallout",
+        "mysteryEncounters/theStrongStuff",
+        "mysteryEncounters/thePokemonSalesman",
+        "mysteryEncounters/anOfferYouCantRefuse",
+        "mysteryEncounters/delibirdy",
+        "mysteryEncounters/absoluteAvarice",
+        "mysteryEncounters/aTrainersTest",
+        "mysteryEncounters/trashToTreasure",
+        "mysteryEncounters/berriesAbound",
+        "mysteryEncounters/clowningAround",
+        "mysteryEncounters/partTimer",
+        "mysteryEncounters/dancingLessons",
+        "mysteryEncounters/weirdDream",
+        "mysteryEncounters/theWinstrateChallenge",
+        "mysteryEncounters/teleportingHijinks",
+        "mysteryEncounters/bugTypeSuperfan",
+        "mysteryEncounters/funAndGames",
+        "mysteryEncounters/uncommonBreed",
+        "mysteryEncounters/globalTradeSystem",
+        "mysteryEncounters/theExpertPokemonBreeder",
+        "mysteryEncounterMessages",
+      ],
+      detection: {
+        lookupLocalStorage: "prLang",
+      },
+      debug: Number(import.meta.env.VITE_I18N_DEBUG) === 1,
```

Comment:
```suggestion
      debug: +(import.meta.env.VITE_I18N_DEBUG) === 1,
```
didn't check if biome wants this to not have parenthesis, but we should never use Number(...). Just use `+`

Reply:
The conversion isn't necessary anyway, just do `=== "1"`.

Reply:
```suggestion
      debug: import.meta.env.VITE_I18N_DEBUG === "1",
```